src/test/java/com/google/api/codegen/testdata/php/php_longrunning.baseline

```diff
@@ -772,7 +782,10 @@ class OperationsClientTest extends GeneratedTest
      */
     private function createClient(array $options = [])
     {
-        return new OperationsClient($options);
+        return new OperationsClient($options + [
+            'serviceAddress' => '',
+            'scopes' => [],
```

Comment:
It looks like this, and the samples need to be updated. The scopes are passed in via `credentialsConfig` now.